+++
title = "Fast, sensitive and accurate integration of single-cell data with Harmony. *Nature Method*, 2019."
date = 2019-11-01T00:00:00

# Authors. 
authors = ["Korsunsky, I.", "Millard, N.", "Fan, J.", "Slowikowski, K.", "**Zhang, F.**", "Wei, K.", "Baglaenko, Y.", "Brenner, M.", "Loh, P.", "Raychaudhuri, S."]

# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference paper
# 2 = Journal article
# 3 = Manuscript
# 4 = Report
# 5 = Book
# 6 = Book section
publication_types = ["2"]

# Publication name and optional abbreviated version.
publication = "In *Nature Method*"
publication_short = ""

# Abstract.
abstract = "The emerging diversity of single-cell RNA-seq datasets allows for the full transcriptional characterization of cell types across a wide variety of biological and clinical conditions. However, it is challenging to analyze them together, particularly when datasets are assayed with different technologies, because biological and technical differences are interspersed. We present Harmony (https://github.com/immunogenomics/harmony), an algorithm that projects cells into a shared embedding in which cells group by cell type rather than dataset-specific conditions. Harmony simultaneously accounts for multiple experimental and biological factors. In six analyses, we demonstrate the superior performance of Harmony to previously published algorithms while requiring fewer computational resources. Harmony enables the integration of ~10e6 cells on a personal computer. We apply Harmony to peripheral blood mononuclear cells from datasets with large experimental differences, five studies of pancreatic islet cells, mouse embryogenesis datasets and the integration of scRNA-seq with spatial transcriptomics data."

# Summary
summary = ""

# Digital Object Identifier (DOI)
doi = "10.1038/s41592-019-0619-0"

# Is this a featured publication? (true/false)
featured = true

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = []

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["deep-learning"]` references 
#   `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects = ["internal-project"]

# Links (optional).
url_pdf = "https://www.ncbi.nlm.nih.gov/pubmed/?term=Fast%2C+sensitive+and+accurate+integration+of+single-cell+data+with+Harmony"
# url_preprint = "https://www.biorxiv.org/content/biorxiv/early/2018/11/05/461954.full.pdf"
url_code = "https://github.com/immunogenomics/harmony"
# url_dataset = "#"


# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
# links = [{name = "Pubmed Link", url = ""}]

# Does this page contain LaTeX math? (true/false)
math = true

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
[image]
  # Caption (optional)
  # caption = "Image credit: [**Unsplash**](https://unsplash.com/photos/pLCdAaMFLTE)"

  # Focal point (optional)
  # Options: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight
  # focal_point = ""
  
# Featured image
# Place your image in the `static/img/` folder and reference its filename below, e.g. `image = "example.jpg"`.
[header]
image = "headers/harmony.png"
# caption = "My caption :smile:" 

+++


